app/src/main/java/com/codingblocks/cbonlineapp/database/HBRankDao.kt

```diff
@@ -0,0 +1,12 @@
+package com.codingblocks.cbonlineapp.database
+
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Query
+import com.codingblocks.cbonlineapp.database.models.HBRankModel
+
+@Dao
+interface HBRankDao :BaseDao<HBRankModel> {
+    @Query("SELECT * FROM HBRankModel ORDER BY id DESC LIMIT 1")
```

Comment:
No need order as there will be only one

Reply:
My approach was to fetch a newly added row